client/components/horizontal-list/index.tsx

```diff
@@ -3,14 +3,24 @@
 /**
  * External dependencies
  */
+import React from 'react';
 import { List } from '@woocommerce/components';
 
 /**
  * Internal dependencies.
  */
 import './style.scss';
 
-const HorizontalList = ( props ) => {
+interface HorizontalListItem {
+	title: string;
+	content: string | React.ReactNode;
+}
+
+interface Props {
+	items: ( false | HorizontalListItem )[];
+}
```

Comment:
Let's add doc comments for the props too, e.g.

```ts
interface HorizontalListItem {
  /**
   * The title for the item, will be displayed on hover. // Side note, I don't actually know what the title is for.
   */
  title: string;

  /**
   * The content that will be displayed in the list for this item.
   */
  content: string | React.ReactNode;
}
```

And the same for the props.

Reply:
Fixed in [c8765a4](https://github.com/Automattic/woocommerce-payments/pull/6249/commits/c8765a449577e2ca324a8a6b551232de9dad07bc).